### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by tudorpintea999**
### Purpose or design rationale of this PR

*This PR's scope is to fix some of the typos that I recently found within the documentation and to make it more readable*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 ## Directory Structure
 
 <pre>
-├── <a href="./bridge-history-api/">bridge-history-api</a>: Bridge history service that collects deposit and withdraw events from both L1 and L2 chain and generates withdrawal proofs
+├── <a href="./bridge-history-api/">bridge-history-api</a>: Bridge history service that collects deposit and withdraws events from both L1 and L2 chains and generates withdrawal proofs
```
Comment (Thegaram) [inline README.md]: Should be "deposit and withdraw events"

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```
Comment (Thegaram) [inline CODE_OF_CONDUCT.md]: "Examples ... include"

**Review by Thegaram — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tudorpintea999) [reply README.md]: that is true, I modified those, thank you for your feedback !

**Comment by tudorpintea999** [comment]
hello @Thegaram do you need something else from my side on this pr?
Thanks !

**Review by 0xmountaintop — APPROVED** [review]

**Review by georgehao — APPROVED** [review]

**Review by Thegaram — APPROVED** [review]

**Pull request merged**